### ISSUE TITLE
.stack on Errors is claimed to be a Firefox-only API, when it's not

### ISSUE THREAD

**Issue by bzbarsky** [GoogleChrome/confluence#237]
It's in all browsers, and required for web compat.  It's just that some browsers make it an instance property, not a prototype property.

**Comment by foolip** [comment]
Yep, looks like Firefox has `Error.prototype.stack` but nobody else. I don't see `stack` in https://tc39.github.io/ecma262/#sec-properties-of-the-error-prototype-object, is this not defined by any spec?

@mdittmer, do we have an umbrella bug for these instance vs. prototype issues?

**Comment by bzbarsky** [comment]
> is this not defined by any spec?

Correct.  It's just needed for web compat.  ;)

**Comment by mdittmer** [comment]
We do not have an umbrella bug for instance vs. prototype issues, but we do
interpret accessible instances. I think this needs a factory to produce an
instance for inspection.

**Comment by bzbarsky** [comment]
`new Error` works fine.
